src/guide/iroha-2.md

```diff
@@ -3,75 +3,93 @@
 Iroha 2 is a complete re-write of Hyperledger Iroha in Rust. As of writing
 the two projects are developed concurrently.
 
+Here we aspire to outline the differences between two versions of Iroha and
```

Comment:
```suggestion
This section will outline the differences between the two versions of Iroha and
```

Reply:
I think the "here we aspire to" wording is a bit awkward and hints at the fact that we have not made this determination already. If "this section" doesn't work out, I'm ok with different phrasing.